php-frontend/src/test/resources/symbols/symbolTable.php

```diff
@@ -9,7 +9,9 @@
 foreach (array("a", "b") as $key => $val) {}  // $key, $val
 
 function f() {                                // f
+  $global = 42;                               // $global already defined, not yet defined as global
   static $static;                             // $static
+  global $global;                             // $global
   $a = 1;                                     // $a
```

Comment:
define $global globally
unit test $global in the scope of function "f"
validate behavior between "local", "static" and "global" for the same variable name.

Reply:
completed tests and added a new one showing inconsistencies.